sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -2245,6 +2245,70 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
       enddo
      if(debugprint)print*,'sample l',VarName,' = ',1,sh2o(isa,jsa,4)
 
+      IF (NSOIL==9) THEN
```

Comment:
@EricJames-NOAA Please add a change log at the the beginning session of this subroutine.